lens-cube/src/main/java/org/apache/lens/cube/parse/StorageCandidate.java

```diff
@@ -54,9 +62,12 @@
   private final float completenessThreshold;
   @Getter
   private final String name;
+  private final List<Integer> selectIndices = Lists.newArrayList();
```

Comment:
selectIndices ,dimFieldIndices are never read. Do we need them ?  
If not, we can simplify the logic in  updateAnswerableQueriedColumns() which populates these members